src/studio-home/data/thunks.js

```diff
@@ -14,7 +14,7 @@ import {
   fetchLibraryDataSuccess,
 } from './slice';
 
-function fetchStudioHomeData(search, hasHomeData) {
+function fetchStudioHomeData(search, hasHomeData, customParams = {}) {
```

Comment:
What about `queryParams` instead of `customParams`?

Reply:
That sounds like a database query, what about `requestParams`?

Reply:
Sounds good.